installer/install.iss

```diff
@@ -1576,9 +1576,9 @@ end;
 
 // Procedure to create hardlinks for builtins. This procedure relies upon that
 // git-wrapper.exe is already copied to {app}\tmp.
-procedure CopyBuiltin(FileName:String);
+procedure HardlinkOrCopyGit(FileName:String,Builtin:Boolean);
```

Comment:
Shouldn't this be a `;` to separate arguments?

Reply:
Uh oh, you're correct! I *always* forget...